app/views/admin/topical_event_about_pages/edit.html.erb

```diff
@@ -1 +1,23 @@
-<%= render partial: 'form', locals: { page_title: "Edit page about #{@topical_event.name}", url: admin_topical_event_topical_event_about_pages_path(@topical_event, @topical_event_about_page) } %>
+<% content_for :page_title, "Edit page about #{@topical_event.name}" %>
+<% content_for :title, "Edit page about #{@topical_event.name}" %>
+<% content_for :title_margin_bottom, 6 %>
```

Comment:
I think this should be 8 as Nik now want's these to be 50px.

Reply:
will change that